src/main/resources/default/assets/tycho/scripts/datepicker.js

```diff
@@ -0,0 +1,20 @@
+sirius.ready(function() {
+    $('.date-picker').datetimepicker({
+        locale: tycho_current_locale,
+        ignoreReadonly: true,
+        keepInvalid: true,
+        useCurrent: false,
+        showTodayButton: true,
+        icons: {
+            time: 'fa fa-clock-o',
+            date: 'fa fa-calendar',
+            up: 'fa fa-chevron-up',
+            down: 'fa fa-chevron-down',
+            previous: 'fa fa-chevron-left',
+            next: 'fa fa-chevron-right',
+            today: 'fa fa-history',
+            clear: 'fa fa-trash',
+            close: 'fa fa-remove'
+        }
+    });
```

Comment:
Question: Did we have a look whether other alternatives are around nowadays? With this one we will never be able to remove jQuery.